projects/addon-doc/src/components/navigation/navigation.template.html

```diff
@@ -62,21 +62,23 @@
                 </span>
                 <ng-template tuiAccordionItemContent>
                     <div class="t-section">
-                        <ng-container
-                            *ngFor="let item of items[index]; index as subIndex"
-                        >
+                        <ng-container *ngFor="let item of items[index]; index as subIndex">
                             <ng-container
-                                *ngTemplateOutlet="pages; context: {item: item, index: index * 100 + subIndex}"
+                                *ngTemplateOutlet="
```

Comment:
Again, can be moved to higher `ng-container` as two separate inputs.

Reply:
fixed